zkevm-circuits/src/evm_circuit/execution.rs

```diff
@@ -311,8 +311,8 @@ pub struct ExecutionConfig<F> {
     error_oog_code_store: Box<DummyGadget<F, 0, 0, { ExecutionState::ErrorOutOfGasCodeStore }>>,
     error_invalid_jump: Box<ErrorInvalidJumpGadget<F>>,
     error_invalid_opcode: Box<ErrorInvalidOpcodeGadget<F>>,
-    error_depth: Box<DummyGadget<F, 0, 0, { ExecutionState::ErrorDepth }>>,
-    error_contract_address_collision:
+    _error_depth: Box<DummyGadget<F, 0, 0, { ExecutionState::ErrorDepth }>>,
```

Comment:
For example, instead of add underscore prefix, which somehow still make reader confused about this, what do you think we have `deadcode` specificly on this field with explicit `reason`. e.g.

```rust
    #[allow(dead_code, reason = "under active development")]
    error_depth: Box<DummyGadget<F, 0, 0, { ExecutionState::ErrorDepth }>>,
``` 

It both caring about readibility and clean up real unused code for now.